### ISSUE TITLE
Support for Codestar Framework

### ISSUE THREAD

**Issue by sabuz** [FrankM1/radium-one-click-demo-install#20]
Hello!
Would you kindly add support for codestar framework?

**Comment by Ansif** [comment]
Yes, please let me codestar framework becoming popular now a days. So, can you please let me know it'll support in codestar framework?
